insights/static/js/components/bar-chart.js

```diff
@@ -24,17 +59,21 @@ export const barChart = {
                     xAxes: [{
                         gridLines: {
                             display: false,
-                            offsetGridLines: false,
+                            offsetGridLines: true,
                         },
                         offset: true,
                         ticks: {
-                            display: true,
+                          userCallback(value, index, ticks) {
+                            if (value instanceof Date) {
+                              const month = new Intl.DateTimeFormat(language, { month: 'long' }).format(value)
```

Comment:
Nice didn't know `Intl` is now a thing!

Reply:
Ended up removing it as it meant recalculating the date every loop. But it's a cool feature!